src/services/AuthService.js

```diff
@@ -1,19 +1,26 @@
 import http from './HttpService';
 
-const auth = {
-    login: function (userEmail, userPassword) {
-        http.post('/auth/local', {
-            email: userEmail,
-            password: userPassword,
+const Auth = (userEmail, userPassword) => {
```

Comment:
- Let's use `async / await` instead of `.then / .catch`
- Don't export just the login function, we'll have several functions in this service, e.g. for user registration, resetting password... everything related to authentication :)